lib/libc/common/source/time/ctime.c

```diff
@@ -4,6 +4,8 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+#undef _POSIX_C_SOURCE
```

Comment:
I understand why the kludge is necessary. I was hoping to address this particular issue preemptively almost 9 months ago :'(

https://github.com/zephyrproject-rtos/zephyr/pull/67132

Reply:
I'm happy to approve as-is, but @keith-packard, if you feel like we could use the Kconfig, I would be more than happy to pursue that PR again.

Reply:
Up to you; we can revert this change if that happens.